CONTRIBUTING.md

```diff
@@ -352,7 +354,44 @@ In cases where a PR is not worth noting in the release notes, you can also tell
 
 Finally, don't fret about this too much! The Release Drafter configuration and labeling scheme may take some time to fine-tune, and the drafted release notes can always be manually edited before final publication.
 
-## Publishing to npm
+## npm package
+
+### Testing the package locally
+
+1. Go to the root of your local working copy of explorer-1
+   ```bash
+   $ cd path/to/explorer-1
+   ```
+2. Create a symlink from your global `node_modules` directory to the local explorer-1 directory
+   ```bash
+   $ npm link
```

Comment:
Could we remove the prompt characters (throughout these changes)? We don't use them above (unless differentiating from output), and Bash is no longer the default shell on macOS, either, so the ol' dollar sign may be increasingly confusing to new developers.

```suggestion
   npm link
```

Reply:
Sorry to nitpick, but could you do this on the other added code samples below, too?